src/drake/fs.clj

```diff
@@ -16,6 +16,19 @@
 (def drake-ignore "Names of files or directories to be ignored by Drake"
   #{"_logs"})
 
+(defn get-directory
+  [path]
+  (loop [f (fs/file path)]
+    (cond
+     (.isDirectory f) f
+     (nil? f) nil
+     :else (recur (.getParentFile f)))))
+
```

Comment:
This function does nothing for directories, but sometimes we want to look up in what directory a file is, such as for CWD setting purposes when running a script.